.travis.yml

```diff
@@ -3,13 +3,20 @@ dist: bionic
 language: julia
 
 before_install:
+  - pwd
   - sudo apt-get update
   - sudo apt-get -y install python3-virtualenv
   - python3 -m virtualenv myenv -p python3
+  - . myenv/bin/activate
+  - pip install numpy cython pytest transonic pythran
+  - git clone -b master --single-branch https://github.com/hpyproject/hpy
+  - cd hpy
+  - git checkout cac5d02
+  - python setup.py develop
+  - cd $HOME/build/paugier/piconumpy
```

Comment:
A lot of this looks wrong -- e.g. `pwd` looks like it was added for debugging, I doubt `$HOME/build/paugier` is correct in the CI environment, and I don't think `. myenv/bin/activate` is necessary after creating the virtual env the line above.

On the other hand, I don't think Travis is running any more, so perhaps having it more up to date is better than having it be not up to date.

Reply:
The `cd $HOME/build/paugier/piconumpy` was necessary for Travis (with https://github.com/paugier/piconumpy/). I've just removed this file now that the tests and the benchmarks run fine with Github Actions.